_data/collection-index.yml

```
@@ -353,3 +353,8 @@
   contact: https://github.com/ksh5022/devcontainer-features/issues
   repository: https://github.com/ksh5022/devcontainer-features
   ociReference: ghcr.io/ksh5022/devcontainer-features
+- name: GMkonan devcontainer features
+  maintainer: GMkonan
+  contact: https://github.com/GMkonan/feature/issues
+  repository: https://github.com/GMkonan/feature
+  ociReference: ghcr.io/GMkonan/feature
```

```suggestion
  ociReference: ghcr.io/gmkonan/feature
```
Hi 👋  The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

---

Sure! Already updated.